drivers/NFSSR.py

```diff
@@ -273,6 +274,7 @@ def generate_config(self, sr_uuid, vdi_uuid):
         resp['sr_uuid'] = sr_uuid
         resp['vdi_uuid'] = vdi_uuid
         resp['sr_sm_config'] = self.sr.sm_config
+        resp['sr_other_config'] = self.sr.other_config
```

Comment:
Because the NFS mount implementation in nfs.py seems to depend on it now. Note that it's not the nfs.py version here, it's the version after applying your internal patch queue.